lib/processed-listener.js

```diff
@@ -61,14 +51,15 @@ class ProcessedListener extends EventListener {
 		}
 	}
 
-	sendEmail({ userEmail, entity, files }) {
+	async sendEmail({ userEmail, entity, files }) {
 
-		const apiExportValidator = new ApiExportValidator(this);
-		const controller = apiExportValidator.validateController(entity);
+		const controller = this.apiExportValidator.validateController(entity);
 
-		const dataToSend = controller.getFilesPaths(files);
+		const dataToSend = {
+			files: await controller.getFilesPaths(files)
+		};
 
-		return this.mailService.setTo(userEmail)
+		return this.session.getSessionInstance(Mail).setTo(userEmail)
```

Comment:
Este mail así como está no se va a mandar, porque no tiene body.

De todos modos, mejor mandarlo con un template, para queno dependa tanto el package.
Se quedarían los siguientes llamados:
- `setTemplateCode('export')` <-- El code del template a enviar.
- `setData({ ... })` <-- Esto se usa para reemplazar variables en el template
- `setEntity('export)` <-- La entidad es a la que se relaciona el mail, que debería ser export
- `setEntityId(exportId)` <-- Acá habría que meterle el ID del export que estás procesando

El `to`, `subject` y el `body` se definen en el template, y se reemplaza con le data.

Más arriba, agregá `userEmail` y `entity` en el objeto de `dataToSend`, así están disponibles para reemplazar en el template